service/policy/kasregistry/key_access_server_registry.proto

```diff
@@ -365,7 +365,7 @@ message CreateKeyRequest {
   // Required
   KeyMode key_mode = 4 [(buf.validate.field).enum.defined_only = true]; // The mode of the key (e.g., local or external)
   // Required
-  bytes private_key_ctx = 5; // Context or additional data specific to the private key, based on the key provider implementation
+  bytes private_key_ctx = 5 [(buf.validate.field).required = true]; // Context or additional data specific to the private key, based on the key provider implementation
```

Comment:
I think this needs to be optional in the case of a user importing just another KAS's public key. We might want to make the public key required.